ktor-io/api/ktor-io.api

```diff
@@ -295,6 +295,7 @@ public abstract interface annotation class io/ktor/utils/io/PublicAPICandidate :
 }
 
 public final class io/ktor/utils/io/ReaderJob : io/ktor/utils/io/ChannelJob {
+	public final fun flushAndClose (Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
```

Comment:
please change the target branch to 3.2.0 or at least mark it with an @InternalAPI